lecture_4/lecture_4_solution/test/components/Letter.spec.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import Letter from '../../js/components/Letter';
+
+describe('Letter', () => {
+
+  var Wrapper = React.createClass({
```

Comment:
Since this kind of wrapping is currently only in 2 places, I will keep it here for now. I will move it around in the next homework. Maybe I will change it after lecture or before if I get any time